tools/tolua/cocos2dx_studio.ini

```diff
@@ -53,7 +53,7 @@ skip =  *::[^visit$ copyWith.* onEnter.* onExit.* ^description$ getObjectType .*
         ActionObject::[initWithDictionary initWithBinary],
         BaseData::[copy subtract],
         ActionTimelineCache::[getInstance loadActionTimelineFromXML loadAnimationWithDataBuffer createActionWithDataBuffer],
-        ActionTimeline::[setFrameEventCallFunc]
```

Comment:
is it something related with this PR? If not, i think you should create a another branch for it.